### PR TITLE
GhA: Change vedenemo key in build workflow

### PR DESCRIPTION
**Pull request by henrirosten**
Follow-up for PR https://github.com/tiiuae/ghaf/pull/358. This should be merged together with https://github.com/tiiuae/ghaf/pull/358.

### CODE REVIEW

**Review by joinemm — APPROVED** [review]

**Comment by joinemm** [comment]
@henrirosten you need review from someone with write access

**Review by cazfi — APPROVED** [review]

**Pull request merged**
